src/manifolds/Hyperbolic.jl

```diff
@@ -1,58 +1,46 @@
 @doc raw"""
-    Hyperbolic{N} <: Manifold
+    Hyperbolic{N} <: AbstractEmbeddedManifold{AbstractIsometricEmbeddingType}
 
-The hyperbolic space $ℍ^n$ represented by $n+1$-Tuples, i.e. in by
-vectors in $ℝ^{n+1}$ using the Minkowsi metric, i.e.
+The hyperbolic space $ℍ^n$ represented by $n+1$-Tuples, i.e. embedded in the
+[`Lorentz`](@ref)ian manifold equipped with the [`MinkowskiMetric`](@ref)
+$⟨\cdot,\cdot⟩_{\mathrm{M}}$. The space is defined as
 
 ```math
 ℍ^n = \Bigl\{p ∈ ℝ^{n+1} : ⟨p,p⟩_{\mathrm{M}}= -p_{n+1}^2
-  + \displaystyle\sum_{k=1}^n p_k^2 = -1, p_{n+1} > 0\Bigr\},
+  + \displaystyle\sum_{k=1}^n p_k^2 = -1, p_{n+1} > 0\Bigr\},.
 ```
 
-where $⟨\cdot,\cdot⟩_{\mathrm{M}}$ denotes the [`minkowski_dot`](@ref)
-is Minkowski inner product. The tangent space $T_p ℍ^n$ is given by
+The tangent space $T_p ℍ^n$ is given by
 
 ````math
 T_p ℍ^n := \bigl\{
 X ∈ ℝ^{n+1} : ⟨p,X⟩_{\mathrm{M}} = 0
 \bigr\}.
 ````
-The Minkowski inner product inntroduces the [`MinkowskiMetric`](@ref), which is
-a Riemannian metric on the tangent bundle $T ℍ^n$.
+Note that while the [`MinkowskiMetric`](@ref) renders the [`Lorentz`](@ref) manifold (only)
+pseudo-Riemannian, on the tangent bundle of the Hyperbolic space it induces a Riemannian
+metric. The corresponding sectional curvature is $-1$.
 
 # Constructor
 
     Hyperbolic(n)
 
 Generate the $ℍ^{n} ⊂ ℝ^{n+1}$
 """
-struct Hyperbolic{N} <: Manifold end
+struct Hyperbolic{N} <: AbstractEmbeddedManifold{AbstractIsometricEmbeddingType} end
```

Comment:
Why not `DefaultIsometricEmbeddingType`?

Reply:
It does not inherit exp/log/vector_transport... from its embedding. I thought about that shortly but could not find exp/log... for Lorentzian. So since we have no formulae there is nothing to inherit – that's why its not default. Like sphere – exp/log/... have to be implemented.